### PR TITLE
Issue #1286 - Strings for casting hint.

### PR DESCRIPTION
**Pull request by liuche**
Adding strings to be exported.
### Pull Request checklist

- [ ] This PR includes thorough **tests** or an explanation of why it does not

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -252,4 +252,11 @@ be temporary, and you can try again later.</li>
     <string name="notification_request_desktop_site">Requested desktop version</string>
     <!-- Notification displayed when non-desktop version of the site is requested-->
     <string name="notification_request_non_desktop_site">Returned to non-desktop version</string>
+
+    <!-- Notification displayed to introduce a "YouTube casting" feature (similar to "broadcasting")
+    Users who have the YouTube app on their phone or tablet can use it to play YouTube videos directly on their tv.
```
Comment (mcomella) [inline app/src/main/res/values/strings.xml]: nit: we usually align the comment blocks so this line should move in

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -252,4 +252,11 @@ be temporary, and you can try again later.</li>
...
+
+    %s will be replaced with the icon that matches the button in the YouTube app for this feature.
```
Comment (mcomella) [inline app/src/main/res/values/strings.xml]: Can we inline an icon? Does it need to be a unicode character to do that?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -252,4 +252,11 @@ be temporary, and you can try again later.</li>
...
+    Clicking on that button in the YouTube app will show YouTube's instructions on how to cast. -->
+    <string name="youtube_casting_onboarding">Cast YouTube videos from your phone or tablet. Open YouTube on your device and tap %s to get started.</string>
```
Comment (mcomella) [inline app/src/main/res/values/strings.xml]: I wonder if this name should be more specific? There can be multiple onboarding/hint experiences. Maybe `youtube_casting_onboarding_toast`

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (liuche) [reply app/src/main/res/values/strings.xml]: We can use ImageSpan for that.

**Pull request merged**